package.json

```diff
@@ -7,30 +7,38 @@
     "type": "git",
     "url": "git+https://github.com/nextcloud/eslint-config.git"
   },
+  "scripts": {
+    "lint": "eslint -c index.js *.js",
+    "lint:fix": "eslint -c index.js --fix *.js"
+  },
```

Comment:
Why is this here? It doesn't serve any purpose, no? :p

Reply:
It does, but it does not belong to this pull request ;-)
Removed that changes from this PR

Reply:
Makes more sense! :grin: